backend/src/services/integration-auth/integration-sync-secret.ts

```
@@ -452,7 +452,9 @@ const syncSecretsAWSParameterStore = async ({
   accessId: string | null;
   accessToken: string;
 }) => {
-  if (!accessId) return;
+  let response: { isSynced: boolean; syncMessage: string } | null = null;
+
+  if (!accessId) return null;
```

let's throw an error here instead of returning null?